<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Results</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="timelineContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="TimelineWidget" name="m_timeline" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QScrollBar" name="m_verticalScrollBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QScrollBar" name="m_horizontalScrollBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graphContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="GraphWidget" name="m_gpuGraph" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GraphWidget" name="m_cpuGraph" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="m_table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimelineWidget</class>
   <extends>QWidget</extends>
   <header>timelinewidget.h</header>
   <container>1</container>
   <slots>
    <signal>horizontalScrollValueChanged(int)</signal>
    <signal>verticalScrollValueChanged(int)</signal>
    <signal>horizontalScrollMaxChanged(int)</signal>
    <signal>verticalScrollMaxChanged(int)</signal>
    <signal>jumpToCall(int)</signal>
    <signal>selectedTime(int64_t,int64_t)</signal>
    <signal>selectedProgram(unsigned)</signal>
    <signal>selectedNone()</signal>
    <slot>setHorizontalScrollValue(int)</slot>
    <slot>setVerticalScrollValue(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
   <container>1</container>
   <slots>
    <signal>viewChanged(int,int)</signal>
    <signal>selectedTime(int64_t,int64_t)</signal>
    <signal>selectedProgram(unsigned)</signal>
    <signal>selectedNone()</signal>
    <slot>changeView(int,int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_horizontalScrollBar</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>m_timeline</receiver>
   <slot>setHorizontalScrollValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_verticalScrollBar</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>m_timeline</receiver>
   <slot>setVerticalScrollValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>horizontalScrollValueChanged(int)</signal>
   <receiver>m_horizontalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>verticalScrollValueChanged(int)</signal>
   <receiver>m_verticalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>horizontalScrollMaxChanged(int)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>setHorizontalScrollMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>verticalScrollMaxChanged(int)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>setVerticalScrollMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuGraph</sender>
   <signal>viewChanged(int,int)</signal>
   <receiver>m_gpuGraph</receiver>
   <slot>changeView(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>687</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gpuGraph</sender>
   <signal>viewChanged(int,int)</signal>
   <receiver>m_cpuGraph</receiver>
   <slot>changeView(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>687</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_table</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>tableDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuGraph</sender>
   <signal>selectedTime(int64_t,int64_t)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectTime(int64_t,int64_t)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gpuGraph</sender>
   <signal>selectedTime(int64_t,int64_t)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectTime(int64_t,int64_t)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>selectedTime(int64_t,int64_t)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectTime(int64_t,int64_t)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuGraph</sender>
   <signal>selectedProgram(unsigned)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectProgram(unsigned)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gpuGraph</sender>
   <signal>selectedProgram(unsigned)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectProgram(unsigned)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>selectedProgram(unsigned)</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectProgram(unsigned)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cpuGraph</sender>
   <signal>selectedNone()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectNone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gpuGraph</sender>
   <signal>selectedNone()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectNone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_timeline</sender>
   <signal>selectedNone()</signal>
   <receiver>ProfileDialog</receiver>
   <slot>selectNone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>jumpToCall(int)</signal>
  <slot>setVerticalScrollMax(int)</slot>
  <slot>setHorizontalScrollMax(int)</slot>
  <slot>tableDoubleClicked(QModelIndex)</slot>
  <slot>selectTime(int64_t,int64_t)</slot>
  <slot>selectProgram(unsigned)</slot>
  <slot>selectNone()</slot>
 </slots>
</ui>
